src/CachePool.php

```diff
@@ -466,9 +466,17 @@ protected function getAllKeys(): iterable
     protected function selectColumn(string $query, string $columnName, array $args = []): iterable
     {
         $query = $this->prepareQuery($query, $args);
-        $results = $this->wpdb->get_col($query, $columnName);
+        $results = $this->wpdb->get_results($query, ARRAY_A);
+
+        $column = [];
+        foreach ($results as $row) {
+            $value = array_key_exists($columnName, $row)
+                ? $row[$columnName]
+                : null;
+            $column[] = $value;
+        }
```

Comment:
You've just invented spanish vax here :)

Reply:
What's "spanish vax"?

Reply:
...English says: invented the wheel

Reply:
Gotcha. Well, I saw your simplifying patch, and LGTM 👍 Gonna go out next release.